modules/olm-operatorhub-architecture.adoc

```diff
@@ -15,36 +21,66 @@ interface, it is only used directly by those who are creating their own
 Operators.
 ====
 
-[id="olm-operatorhub-arch-operatorsource_{context}"]
-== OperatorSource
-
-For each Operator, the OperatorSource is used to define the external data store
-used to store Operator bundles. A
-link:https://github.com/operator-framework/operator-marketplace/blob/master/deploy/examples/community.operatorsource.cr.yaml[simple OperatorSource]
-includes:
-
-[cols="2a,8a",options="header"]
-|===
+[NOTE]
+====
+While OperatorHub no longer uses CatalogSourceConfig resources, they are still
+supported in {product-title}.
+====
 
-|Field
-|Description
+[id="olm-operatorhub-arch-operatorhub_crd_{context}"]
+== OperatorHub CRD
 
-|`type`
-|To identify the data store as an application registry, `type` is set to `appregistry`.
+The OperatorHub CRD can be used to change the state of the default
+OperatorSources provided with OperatorHub on the cluster from enabled to
```

Comment:
I'd rework this sentence to include "you can" and remove "vice versa."

Do you use these CRs in a procedure elsewhere? I don't think I'd duplicate them here.